### PR TITLE
Refactor webhooks

### PR DESCRIPTION
**Pull request by relekang**
#### Github
- [x] Pinge events
- [x] Push events
- [x] Tag push events
- [x] Pull request events
- [x] Issue comment events

### CODE REVIEW

**Comment by relekang** [comment]
retest now please

File: frigg/webhooks/helpers.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+            self.project.start_build({
+                'branch': self.branch,
+                'sha': self.hash,
+                'author': self.author,
+                'pull_request_id': self.pull_request_id,
+                'message': self.message
+            })
+
+
+class GithubEvent(Event):
```
Comment (frecar) [inline frigg/webhooks/helpers.py]: what about puttin each class in it's own file, githubevent.py etc ?

Comment (relekang) [inline frigg/webhooks/helpers.py]: :+1:

**Comment by frecar** [comment]
:shipit:  very good :D

**Pull request merged**